scripts/ai/mcp/support/uv_resolver.ps1

```
@@ -142,6 +146,27 @@ function Resolve-BioetlUvxBin {
+            [char[]]@($pathSeparator, ';', ':'),
+            [StringSplitOptions]::RemoveEmptyEntries
+        )
+    }
+    foreach ($entry in $pathEntries) {
+        foreach ($name in @("uvx.exe", "uvx.cmd", "uvx.ps1", "uvx")) {
+            try {
+                $candidates += [System.IO.Path]::Combine($entry.Trim(), $name)
+            }
+            catch {
+                # Ignore unusable PATH entries.
+            }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)